docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb

```diff
@@ -287,26 +320,26 @@
     }
    ],
    "source": [
-    "if not os.path.exists('/root/7.png'):\n",
-    "    !wget -P /root/ https://ascend-tutorials.obs.cn-north-4.myhuaweicloud.com/tinyms-test-pics/numbers/7.png\n",
+    "if not os.path.exists('/home/jovyan/7.png'):\n",
+    "    !wget -P /home/jovyan https://ascend-tutorials.obs.cn-north-4.myhuaweicloud.com/tinyms-test-pics/numbers/7.png\n",
     "else:\n",
     "    print('7.png already exists')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "convertible-timer",
+   "id": "40b5e41a",
    "metadata": {},
    "source": [
     "#### 6.2 List servables\n",
     "\n",
-    "使用`list_servables`函数检查当前后端的serving模型"
+    "使用`list_servables`函数检查当前后端的serving模型,如果这边报错，在root下download_dataset('mnist', '/root'),然后从头运行一遍。"
```

Comment:
Please modify punctuation formatting problems

Reply:
ok i will perfect it